Model/Checkout/ShippingMethods.php

```diff
@@ -11,32 +12,70 @@
 class ShippingMethods implements ShippingMethodsInterface
 {
     /**
-     * @param mixed $deliveryOptions
+     * @var \Magento\Checkout\Model\Session
+     */
+    private $session;
+
+    /**
+     * ShippingMethods constructor.
+     *
+     * @param \Magento\Checkout\Model\Session $session
+     */
+    public function __construct(Session $session)
+    {
+        $this->session = $session;
+    }
+
+    /**
+     * @param array[] $deliveryOptions indexed array holding 1 deliveryOptions object
      *
-     * @return mixed[]
+     * @return array[]
      * @throws Exception
      */
-    public function getFromDeliveryOptions($deliveryOptions)
+    public function getFromDeliveryOptions(array $deliveryOptions): array
     {
-        if (! $deliveryOptions[0]) {
+        if (! $deliveryOptions) {
```

Comment:
`if (empty($deliveryOptions)) {`

Reply:
🤔  https://wiki.dmp.zone/display/MD/DSR-1%3A+PHP+Standards#DSR-1:PHPStandards-Emptyorexclamationmark

Reply:
Ik stel voor om het terug te veranderen naar hoe het was, en alleen een verklaring in het docblock te zetten